src/identity.js

```diff
@@ -1784,31 +1700,28 @@ export default function Identity(mpInstance) {
 
     this.sendUserIdentityChangeEvent = function(
         newUserIdentities,
-        method,
         mpid,
-        prevUserIdentities
+        currentUserIdentities
     ) {
-        var currentUserInMemory, userIdentityChangeEvent;
-
-        if (!mpid) {
-            if (method !== Modify) {
-                return;
-            }
-        }
```

Comment:
I'm a little nervous to remove this code, even if it doesn't break any tests given the code was not commented at all before.  Maybe we can add this code back in, and add a go link for trying to figure out what is going on with this code block.

Reply:
I think the reason we removed it is because the original usage was possibly incorrect. For example:

```javascript
self.sendUserIdentityChangeEvent(
    newIdentitiesByName,
    method,
    identityApiResult.mpid,
    method === Modify
        ? previousUIByNameCopy
        : incomingMpidUIByNameCopy
);
```

Given that, we'd also have to restore the original signature as well.